src/rulesets/properties.ts

```diff
@@ -176,7 +172,7 @@ export const rules = {
         if (specItem.type === "object") {
           if (!specItem.properties)
             expect.fail("object schema is missing 'properties'");
-          if (specItem.hasOwnProperty("items"))
+          if (Object.prototype.hasOwnProperty.call(specItem, "items"))
```

Comment:
Linter didn't like it:
```
  175:24  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
```

Reply:
I've checked the tests, and they are not touching the condition. Test coverage is not good here, it's a bit worrying.

However I'm confident on the JS change: 
![image](https://user-images.githubusercontent.com/347098/162033969-e9efd287-13c5-445c-a119-cd92a3616846.png)

I would suggest making a test case that touches the condition, that will be the best.

Reply:
I made an issue to add a test coverage here https://github.com/snyk/sweater-comb/issues/236